gizmo.py

```diff
@@ -0,0 +1,21 @@
+def hello(name,country="Finland"):
+    print("Hello {0}, how are things in {1}?".format(str(name),str(country)))
+
+def spell(input_str):
+    result=input_str[0]
+    for i in input_str[1:]:
+        result+="."+i
+        
+    return print(result)
+
+def relative_path(subject_identifiers,relpath='./subjects/moc_recording'):
+    names=[]
+    relpath='./subjects/mock_recording_'
+    for i in subject_identifiers:
+        res=relpath+"_"+i+".rec"
```

Comment:
This works, but there is a much better way. At you convenience, check out pythons amazing "f-strings": https://realpython.com/python-f-strings/

Reply:
Thank you. I updated the exercises. I have to say this is real fun, thanks for the training!